ci/functional/required_packages.sh

```diff
@@ -30,7 +30,9 @@ elif [[ $distro = el* ]] || [[ $distro = centos* ]] ||
           hdf5-$openmpi-tests          \
           hdf5-vol-daos-$openmpi-tests \
           hdf5-vol-daos-mpich-tests    \
-          MACSio-mpich                 \
+          simul-mpich                  \
+          simul-$openmpi               \
```

Comment:
These should NOT have gone into this file.  This whole list of packages and this file in fact are deprecated.  These should be specified as `Requires:` to a tests RPM.